pkg/sql/distsqlrun/columnbackfiller.go

```diff
@@ -223,6 +230,11 @@ func (cb *columnBackfiller) runChunk(
 		updateValues := make(tree.Datums, len(cb.updateExprs))
 		b := txn.NewBatch()
 		rowLength := 0
+		iv := &sqlbase.RowIndexedVarContainer{
+			Cols:    tableDesc.Columns,
```

Comment:
Are you sure this is the correct set of columns? I ask because it's not clear here whether we need to add the mutation columns as well?

Reply:
I suppose is it possible to add two columns in the same statement, where one has a default value and the second is a computed column that depends on the first?

Reply:
I added a test for the default + computed column in the same ALTER that fails (as expected).